### ISSUE TITLE
GitLab Repository Integration: Plugin Not Adding committer details to UI

### ISSUE THREAD

**Issue by anibmurthy** [ojacques/mkdocs-git-committers-plugin-2#28]
I want to express my appreciation for creating this fantastic plugin!

I'm assuming that this plugin can also be utilized for GitLab repositories.

I made an attempt to apply this plugin to my GitLab repository, but unfortunately, I haven't observed any changes. I noticed that the `.cache/plugins/git-committers` file is generated with an empty `authors` list. I suspect that this might be the reason why no content is being added to the user interface. Below is the plugin configuration specified in `mkdocs.yaml`:

```
plugins:
  - git-committers:
      repository: anibmurthy/mkdocs
      enterprise_hostname: gitlab.com
      branch: main
```

**Comment by ojacques** [comment]
> I want to express my appreciation for creating this fantastic plugin!

Oh, thanks!!

> I'm assuming that this plugin can also be utilized for GitLab repositories.

Unfortunately - for now - not the current version. It will only work for GitHub repositories. I was thinking of updating for GitLab, but its not very high in my list of priorities (I accept GutHub sponsorship though 😇 ).

The other aspect is that this plugin - when used with materials for mkdocs - does require the materials insiders version. 

And thanks for the detailed report.

**Comment by ojacques** [comment]
Fixed in [2.1.0](https://github.com/ojacques/mkdocs-git-committers-plugin-2/releases/tag/2.1.0). 

If you use mkdocs-material-insiders, you will see an icon and mention of GitHub, even if the repo is on GitLab. I will open an issue with @squidfunk.

![image](https://github.com/ojacques/mkdocs-git-committers-plugin-2/assets/5385290/accb29e1-a47c-4422-948a-36906dcf159a)
